src/tools/commands/bootstrap-config.ts

```diff
@@ -0,0 +1,73 @@
+import * as fs from 'fs';
+
+import type { CommandModule } from 'yargs';
+
+import { ConfigurationWireHelper } from '../../configuration-wire/configuration-wire-helper';
+import { process } from '../node-shim';
+
+export const bootstrapConfigCommand: CommandModule = {
+  command: 'bootstrap-config',
+  describe: 'Generate a bootstrap configuration string',
+  builder: (yargs) => {
+    return yargs.options({
+      key: {
+        type: 'string',
+        description: 'SDK key',
+        alias: 'k',
+        default: process.env.EPPO_SDK_KEY,
+      },
+      sdk: {
+        type: 'string',
+        description: 'Target SDK name',
+        default: 'android',
+      },
+      'base-url': {
+        type: 'string',
+        description: 'Base URL for the API',
+      },
+      output: {
+        type: 'string',
+        description: 'Output file path',
+        alias: 'o',
+      },
+    });
+  },
+  handler: async (argv) => {
+    if (!argv.key) {
+      console.error('Error: SDK key is required');
+      console.error('Provide it either as:');
+      console.error('- Command line argument: --key <sdkKey> or -k <sdkKey>');
+      console.error('- Environment variable: EPPO_SDK_KEY');
+      process.exit(1);
+    }
+
+    try {
+      const helper = ConfigurationWireHelper.build(argv.key as string, {
+        sdkName: argv.sdk as string,
+        sdkVersion: 'v5.0.0',
```

Comment:
I don't love that this is hard-coded, but it seems reasonable. Pulling from Package.json or passing it around wouldn't be great.

Reply:
removed hard-coding. we should be able to get by without specifying the version. Every mobile SDK version that will support configWire will also support obfuscated config (and inferring whether it is obfuscated).